Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeyTable+consistentReadQuery.swift

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+//  DynamoDBCompositePrimaryKeyTable+consistentReadQuery.swift
+//  SmokeDynamoDB
+//
+
+import Foundation
+import SmokeHTTPClient
+import DynamoDBModel
+import NIO
+
+public extension DynamoDBCompositePrimaryKeyTable {
```

Comment:
What is the purpose of this extension? On the protocol we have query APIs that specify a consistentRead parameter as input. If the purpose here is to provide a variants of the APIs with the default value of consistentRead, we can do this more explicitly by not having this parameter as an input to these functions and specifying the default value explicitly to the inner function.

One of the problems with what you currently have is, if the consistentRead parameter is specified, the complier can't distinguish between which function to actually call and the code would fail to compile.

Reply:
Yes, my purpose is to set a defualt value for consistentRead so that it won't break anything. Do you mean we can set a default true value to the inner function as DynamoDBCompositePrimaryKeyTable only used for non-GSI case?